cpu/cc2538/dev/uart.c

```diff
@@ -136,7 +136,9 @@
  * Baud rate defines used in uart_init() to set the values of UART_IBRD and
  * UART_FBRD in order to achieve the configured baud rates.
  */
-#define UART_CLOCK_RATE       16000000 /* 16 MHz */
+
+#define UART_CLOCK_RATE			(SYS_CTRL_32MHZ/(1 << SYS_CTRL_SYS_DIV))
+
```

Comment:
- Remove the useless empty lines.
 - Keep the original alignment.
 - Use the I/O clock (confirming test pending).
 - Use the existing definition:
```
#define UART_CLOCK_RATE       SYS_CTRL_IO_CLOCK
```